src/shared/icons/checkbox/icons-16/Check.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="11" viewBox="0 0 12 11" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think this SVG is actually used anywhere? It wasn't added to `icons/index.ts` and in the component you use the old `check` icon

Reply:
I probably forgot to replace the old one along the way, fixed now 👍

Reply:
I probably forgot to replace the old one along the way, fixed now 👍